app/models/user.rb

```diff
@@ -26,7 +26,7 @@ def generate_uuid
   end
 
   def update_user_pias_infos
-    user_pias = UserPia.where({user: id})
+    user_pias = UserPia.where({user_id: id})
```

Comment:
You also need to remove the brackets
```suggestion
    user_pias = UserPia.where(user_id: id)
```